content/applications/sales/sales/products_prices/prices/currencies.rst

```diff
@@ -1,49 +1,239 @@
-=================================
-How to sell in foreign currencies
-=================================
+==================
+Foreign currencies
+==================
+
+With Odoo, pricelists can be used to manage prices in a number of foreign currencies. Specifically,
+Odoo has the ability to work with 167 total currencies. In order to use multiple currencies in Odoo
+*Sales*, the *Accounting* application **must** be installed.
+
+Settings
+========
 
-Pricelists can also be used to manage prices in foreign currencies.
+Once the *Accounting* app is installed, foreign currencies can be viewed, edited, and added to the
+database. To do that, navigate to :menuselection:`Accounting app --> Configuration --> Settings`,
+scroll to the :guilabel:`Currencies` section, and locate the :guilabel:`Main Currency` setting.
 
-* Check *Allow multi currencies* in :menuselection:`Invoicing/Accounting --> Settings`.
-  As admin, you need *Adviser* access rights on Invoicing/Accounting apps.
+.. image:: currencies/main-currency-setting-page.png
+   :align: center
+   :alt: How the main currency feature appears on settings page in Odoo Accounting.
+
+Odoo automatically sets the :guilabel:`Main Currency` as the currency of the country the company is
+based in.
 
-* Create one pricelist per currency. A new *Currency* field shows up in pricelist setup form.
+To change the main currency of the company, select the drop-down in the :guilabel:`Currency` field,
+and select the desired currency. Click :guilabel:`Save` to save all changes.
 
 .. tip::
-    To activate a new currency, go to :menuselection:`Accounting --> Configuration --> Currencies`,
-    select it in the list and press *Activate* in the top-right corner. Now it will show up in
-    currencies drop-down lists.
+   To ensure currency rates are updated automatically, enable the :guilabel:`Automatic Currency
+   Rates` feature on the *Accounting* :guilabel:`Settings` page (:menuselection:`Accounting app -->
+   Configuration --> Settings --> Currencies section`).
```

Comment:
The active instruction hasn't started yet, so this text is not present on the screen– no guilabels :)
```suggestion
   To ensure currency rates are updated automatically, enable the *automatic currency rates* feature
   on the *Accounting* settings page (:menuselection:`Accounting app --> Configuration --> Settings
   --> Currencies section`).
```